security/rbac/delta_configs/connect-avro-distributed.properties.delta

```diff
@@ -1,37 +1,19 @@
-bootstrap.servers=SASL_PLAINTEXT://localhost:9092
-
-# Can also be SASL_SSL; if so, make sure to change the protocol on the line above as well
+bootstrap.servers=localhost:9092
 security.protocol=SASL_PLAINTEXT
 sasl.mechanism=OAUTHBEARER
-
 sasl.login.callback.handler.class=io.confluent.kafka.clients.plugins.auth.token.TokenUserLoginCallbackHandler
-
 sasl.jaas.config=org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required username="connect" password="connect1" metadataServerUrls="http://localhost:8090";
 
 ## Connector client (producer, consumer, admin client) properties ##
 
 # Allow producer/consumer/admin client overrides (this enables per-connector principals)
 connector.client.config.override.policy=All
 
-# Default producer configs to use for connectors
```

Comment:
I think we can (and should) retain these. The only property that we _don't_ want to set in the worker file is `sasl.jaas.config`, which provides the actual credentials that identify the service account. Since we want each connector to have its own service account, there shouldn't be a default for that.

However, everything else (`security.protocol`, `sasl.mechanism`, and `sasl.login.callback.handler`) just specifies how authentication will happen (but not user credentials), so it's fine to set those as global defaults that every connector will inherit unless explicitly overridden.

Reply:
@C0urante thanks for the explanation and it makes sense.  I've modified the code, please let me know what you think.

Reply:
Looks good!